xl/settings.py

```diff
@@ -240,12 +240,16 @@ def _val_to_str(self, value):
             Turns a value of some type into a string so it
             can be a configuration value.
         """
-        for k, v in TYPE_MAPPING.iteritems():
+        # Special treatment is needed for boolean values, as bool is a
+        # subclass of int.
+        if isinstance(value, bool):
+            return 'B: ' + str(value)
```

Comment:
This change actually fixes a bug which is consistent in Python 3, but still may happen randomly in Python 2.

Reply:
Thanks; I've committed this fix as e677d2fc.